amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/declaration/OasRefParser.scala

```diff
@@ -157,12 +157,11 @@ class OasRefParser(map: YMap,
     }
 
   private def searchRemoteJsonSchema(ref: String, text: String, e: YMapEntry) = {
-    val fullRef = ctx.resolvedPath(ctx.rootContextDocument, ref)
-    ctx.findJsonSchema(fullRef) match {
-      case Some(u: UnresolvedShape) => copyUnresolvedShape(ref, fullRef, e, u)
+    val fileUrl = ctx.resolvedPath(ctx.rootContextDocument, ref)
```

Comment:
Isn't this the fileUrl + fragment of the json entry?

Reply:
true, will adjust naming to make it clear.